EIPS/eip-1690.md

```diff
@@ -0,0 +1,178 @@
+---
+eip: 1690
+title: ERC-1690 - Mortability Standard
```

Comment:
Not sure this title is descriptive enough, but I do not have any good suggestions. Perhaps "Contract Mortability Interface"?

Reply:
```suggestion
title: ERC-1690 - Contract Mortability Standard
```